firstbyte/static/firstbyte/style/main.css

```diff
@@ -1,143 +1,158 @@
 /* Fonts */
 
 body {
-    font-family: 'Open Sans', sans-serif;
+  font-family: "Open Sans", sans-serif;
```

Comment:
I totally support sticking to automatically formatting our code (share your settings?) but I think we should keep it to it's own PR. I'd be happy to just reset the formatting myself and commit here, is that alright with you?

Reply:
Yes that's fine, I'm just using the prettier formatter I think and didn't realize that it had changed that - I can probably adjust my settings to match too though.